faqs/galaxy/datasets_icons.md

```diff
@@ -9,9 +9,14 @@ contributors: [jennaj, garimavs]
 
 Dataset icons and their usage: 
 
-- {% icon galaxy-eye %} **"Eye icon"**: Display data of the job in the browser.
-- {% icon galaxy-pencil %} **"Pencil icon"**: Edit attributes of the job.
-- {% icon galaxy-delete %} **"Trash icon"**: Delete the job.
-- {% icon galaxy-info %} **"Info icon"**: Job details and run information.
-- {% icon galaxy-refresh %} **"Refresh/Rerun icon"**: Run this (selected) job again or examine original submitted form (filled in).
-- {% icon galaxy-bug %} **"Bug icon"**: Review and optionally submit a bug report.
+- {% icon galaxy-eye %} **"Eye icon"**: Display dataset contents.
+- {% icon galaxy-pencil %} **"Pencil icon"**: Edit attributes of dataset metadata: labels, datatype, database.
+- {% icon galaxy-delete %} **"Trash icon"**: Delete the dataset.
+- {% icon fa-save %} **"Disc icon"**: Download the dataset.
+- {% icon link %} **"Copy link"**: Copy link URL to the dataset.
+- {% icon galaxy-info %} **"Info icon"**: Dataset details and job runtime information: inputs, parameters, logs.
+- {% icon galaxy-refresh %} **"Refresh/Rerun icon"**: Run this (selected) job again or examine original submitted form.
+- {% icon fa-bar-chart-o %} **"Visualize icon"**: External display links (UCSC, IGV, NPL, PV); Charts and graphing; Editor (manually edit text).
+- {% icon fa-sitemap %} **"Dataset Map icon"**: Filter the history for related Input/Output Datasets.
+- {% icon fa-sitemap %} **"Selected Dataset Map icon"**: The "root dataset" for Input/Output Map. Click to clear filter.
```

Comment:
We should add these two to `_config.yml` with some nicer alias, `galaxy-map-datasets` maybe?